README.md

```diff
@@ -53,11 +54,52 @@ Run the following commands to prepare the environment and management cluster:
 
 Apply customizations provided by this repo:
 
-**_Optional_**: This step is needed if your mangement cluster is an OpenShift Cluster
+we provide GitOps ways to apply customizations
+
+1. Bootstrap AgoCD instance on management Cluster
+
+**_Note_**: These steps will disable the default argocd instance with setting DISABLE_DEFAULT_ARGOCD_INSTANCE as 'true' and will provision a argocd instance in namespace named as  "gitops". 
+```bash
+
+#login OCP cluster with your token and API server URL, you can skip this step if already connected with OCP cluster 
+oc login  --token=<your_token> --server=https://api.<clustername>.<domain_name>:6443
+
+oc apply -f bootstrap/subscription.yaml
+oc apply -f bootstrap/cluster-rolebinding.yaml
+
+until [[ $(oc get subscriptions.operators.coreos.com openshift-gitops-operator  -n openshift-gitops-operator -o jsonpath='{.status.conditions[0].message}') == 'all available catalogsources are healthy' ]]
+do
+  echo "Waiting for openshift-gitops-operator subscription ready :"
+  oc get subscriptions.operators.coreos.com openshift-gitops-operator  -n openshift-gitops-operator
+  sleep 10
+done
+
+oc apply -f bootstrap/argocd.yaml
+
+ ```
+
+2. Bootstrap management Cluster configurations
+
+ Prerequisites:
+  - Download argocd cli according you OS type from [here](https://github.com/argoproj/argo-cd/releases) 
+  - Install argocd cli on you laptop or server where you will issue the command to create the applicationSet, for Linux server, move the argocd cli to /usr/local/bin/argocd
+  - Login argocd server with username and password
+
+```bash
+curl -L  -o /usr/local/bin/argocd  https://github.com/argoproj/argo-cd/releases/download/v2.11.0/argocd-linux-amd64
+chmod  +x /usr/local/bin/argocd
+
+argocd login openshift-gitops-server-gitops.$(oc get ingress.config.openshift.io cluster --template={{.spec.domain}}) #input argoCD username and password 
+
+argocd appset create bootstrap/managementClusterConfigApplicationSet.yaml
```

Comment:
`oc create -f bootstrap/managementClusterConfigApplicationSet.yaml` should things going. We don't need to have argot CLI installed

Reply:
agreed - let's showcase the power of the OpenShift / k8s integrations